### PR TITLE
Fix/filter by file format

### LINKED ISSUE #2703: Multiple filetypes inside a collections' folder (Must specify valid format error)
## Summary:
I first got this error after I navigated into my content directory using finder on OS X.  Finder added an invisible `.DS_Store` file in my folder and that resulted in a `must specify valid format` error when trying to load that collection in TinaCMS.  After removing the DS_store file, my collection loaded fine. 

I tested again by dropping a PDF file in the content folder, next to my markdown files and got the same result. 

Screen Shot 2022-03-15 at 3 57 49 PM
Screen Shot 2022-03-15 at 3 56 52 PM

## Note:
Possible solution: look at the format for the collection and if there is a format that isn't part of the collection nor the default, ignore it; for cloud portion verify where we are using glob (in the resolver) _and_ where we are doing indexing (in the database)
Test for backwards compatibility
Add verbiage for "you need to specify a format" or something along those lines

### PR DESCRIPTION
**Pull request by logan-anderson**
fixes #2703 in local mode.

### CODE REVIEW

**Review by kldavis4 — COMMENTED** [review]
@logan-anderson I merged the isomorphic bridge yesterday - does that need this update as well?

File: packages/@tinacms/datalayer/src/database/store/filesystem.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export class FilesystemStore implements Store {
     this.rootPath = rootPath || ''
   }
   public async query(
-    queryOptions: StoreQueryOptions
+    _queryOptions: StoreQueryOptions
```
[the remark this answers is not in the document]
Comment (logan-anderson) [reply packages/@tinacms/datalayer/src/database/store/filesystem.ts:30]: It was giving me a linting warning that it was "unused"  when I add the underscore it goes away.

**Comment by logan-anderson** [comment]
@kldavis4 I had to update some of the tests as the behaviour is now changed. Mind taking a look?

**Review by kldavis4 — APPROVED** [review]
👍🏻

**Pull request merged**